cirq-core/cirq/qis/clifford_tableau.py

```diff
@@ -178,12 +179,12 @@ def _str_full_(self) -> str:
 
         return string
 
-    def merged_with(self, second: 'CliffordTableau') -> 'CliffordTableau':
-        """Returns a merged CliffordTableau of self tableau and second tableau.
+    def then(self, second: 'CliffordTableau') -> 'CliffordTableau':
+        """Returns a composed CliffordTableau of this tableau and the second tableau.
 
-        The meaning of merge two clifford tableau is the corresponding unitary operation
-        of merged clifford tableau is equal to (up to global phase) the composed unitary
-        operation of self tableau and the one of second tableau.
+        Then composed tableau is equal to (up to global phase) the composed
+        unitary operation of the two tableaux, i.e. equivalent to applying the unitary operation
+        this CliffordTableau then applying the second one.
```

Comment:
nit: applying the unitary operation **of** this CliffordTableau

Reply:
Done.